tests_e2e/playwright/conftest.py

```diff
@@ -0,0 +1,44 @@
+import logging
+import os
+
+import dotenv
+import pytest
+from navigator import Navigator
+
+logger = logging.getLogger()
+
+
+@pytest.fixture(scope="session")
+def _navigator_session(browser_name):
+    """Set up a navigator instance, login with username/password, start
+    a server. Teardown when session is complete.
+    Do not use this for individual tests, use `navigator` fixture
+    for tests."""
+    dotenv.load_dotenv()
+    # try/except added here in attempt to reach teardown after error in
+    # order to close the browser context which will save the video so I debug
+    # the error.
+    try:
+        nav = Navigator(
+            nebari_url=os.environ["NEBARI_FULL_URL"],
+            username=os.environ["USERNAME"],
+            password=os.environ["PASSWORD"],
+            headless=False,
+            browser=browser_name,
+            auth="password",
+            instance_name=os.environ["INSTANCE_NAME"],
+            video_dir="videos/",
+        )
+        nav.login_password()
```

Comment:
Developing locally, I don't have a domain. Meaning, I'm just using an IP, e.g. `172.18.1.100` as domain. In this case, the login fails, because playwright can't connect. It needs to be `https://172.18.1.100`. This seems to be handled automatically for proper domains, but we can't guarantee this.

Reply:
I dont understand where the error is here. Is the error because this code is trying to access an https that is unavailable? Or is this code injecting https and it shouldnt? Or does the nebari login require https but you don't have it?

Reply:
For me `self.page.goto(self.nebari_url)` failed if `self.nebari_url` was just an IP. I needed to prefix that manually with `f"https://{self.nebari_url}"` to get it to work.

Reply:
The `https` is always required. I'll add a note to the docs